test/Linter/Yaml/YamlLinterTest.php

```diff
@@ -67,7 +67,12 @@ function it_should_validate_yaml_for_syntax_errors($fixture, $errors)
     function it_should_be_able_to_handle_object_support()
     {
         $this->linter->setObjectSupport(true);
-        $fixture = YamlLinter::supportsFlags() ? 'object-support.yml' : 'object-support-old.yml';
+
+        $fixture = 'object-support-old.yml';
+        if (YamlLinter::supportsFlags()) {
```

Comment:
It's better to move these 3 different test scenario's to 3 different tests and skip the once that do not apply instead.

Reply:
Will update this, later on.